src/renderer/state.ts

```diff
@@ -831,6 +914,33 @@ export class AppState {
     window.location.hash = hash;
   }
 
+  /**
+   * Maps the InstallState to Versionstate enum
+   * This lives temporary here until fiddle-core exports
+   * the same type.
```

Comment:
Is this dependent on a new release of fiddle-core, or does this still need to be changed in fiddle-core?

Reply:
I added this because it this can be integrated in fiddle-core and then imported here.
(We are passing the states as string in fiddle-core, [one such example](https://github.com/electron/fiddle-core/blob/e22ae4d453ba21c5d40f47b153105f632a624c9a/src/installer.ts#L117)).

Reply:
Local branch changes - (https://github.com/electron/fiddle-core/compare/main...aryanshridhar:fiddle-core:Add-VersionStateEnum?expand=1)

Reply:
I have opened up a pr for it - https://github.com/electron/fiddle-core/pull/33
Once this is merged, we don't need to have this extra check.